draft-ietf-rats-eat.md

```diff
@@ -1786,6 +1800,54 @@ quality random number generators are readily available as they are
 implemented in commonly used CPU hardware.
 
 
+# EAT Relation to IDevID
+
+This section describes several distinct ways in which an IEEE IDevID [IDevID] relates to EAT, particularly to UEID and SUEID.
+
+
+## Signed Device IDs
+
+A unique device identifier is of little use in security-related applications unless it is signed.
+Without signing, it is trivial for attackers to forge a device identifier.
+
+In a way IDevID and EAT are competitors.
+They both provide signed unique device identifiers.
+
+EAT provides the unique identifier as a single small data item, the UEID and SUEID.
+For IDevID the unique identifier is through and X.500 distinguished name, perhaps constructed from the certificate chain, 
+some how derived from the key material.
+
+This document argues EAT is the more general solution.
+It allows an open-ended set of claims about the device to be signed and conveyed in addition to just the identifier.
+It separates the signing scheme from the identification scheme.
+EAT even allows use of external mechanisms to secure the device identifier and additional claims, for example use of TLS with UCCS.
+
+
+## Permanence
+
+In terms of permanence, an IDevID is similar to a UEID in that they do not change over the life of the device.
+They cease to exist only when the device is destroyed.
+
+An SUEID is similar to an LDevID.
+They change on device life-cycle events.
+
+[IDevID] describes much of this permanence as resistant to attacks that seek to change the ID.
+Since EAT is for use on a range of device from high-security device to low-security devices, permanence can't be described this way here.
+Also, EAT is a protocol document that doesn't specify an implementation.
+By contrast, IDevID is primarily an implementation document, not a protocol document.
```

Comment:
I don't understand these two claims.  Also IDevID is not a document.

Reply:
Hopefully rewording in first part of this appending addresses this.